MatrixSDK/Crypto/Verification/MXKeyVerificationManagerV2.swift

```diff
@@ -399,6 +369,25 @@ class MXKeyVerificationManagerV2: NSObject, MXKeyVerificationManager {
     
     // MARK: - Verification requests
     
+    func requestVerificationByToDevice(
+        withUserId userId: String,
```

Comment:
Is this `userId` supposed to be used below instead of the session one? 🤔

Reply:
Not sure I understand the question. The method serves to verify any user, incl self, but the ffi  interface has explicitely `requestSelfVerification` vs `requestVerification(user:...)`. So if the passed-in userId matches the session, we use self-verification.

Reply:
Ups sorry, my bad. I completely missread that being as an `if **let** userId == session?.myUserId`, please disregard.